packages/cli/README.md

```diff
@@ -273,6 +273,30 @@ bubblewrap fingerprint generateAssetLinks <flags>
 Flags:
  - `--output=<filename>`: path from where to load the project configuration.
 
+## `play`
+
+:warning: This is an experimental feature.
+
+Manages the artifacts for your Google Play Project.
+
+Usage:
+
+```
+bubblewrap play --serivceAccountFile="/path/to/serivce/account"  --retain 86 --publish
+```
+
+This tool requires a service account file to work correctly. Please see [this documentation](https://github.com/chromeos/pwa-play-billing#setup-a-service-account) for setting up a service acocunt.
+
+Options:
+  - `--publish`: publishes the prebuilt file to the Google Play Store under the internal track or specificed track.
+  - `--serviceAccountJsonFile`: sets the service acocunt json file location in the twa-manifest.
```

Comment:
nit: typo on "account".

So this command does two things if `serviceAccountJsonFile` is present - it both publishes and saves something to the TWA manifest. Will we ever want to save the thing to the TWA manifest separately? I'm a little wary of commands having side effects of both "override this value" and "write the overwritten value to something".

Reply:
Also, are there multiple valid formats for the service account file? If not then we can probably get rid of the `json`.

Reply:
Fixed the account typo.

This never overrides the value, but always sets the serviceAccountFile in the twa-manifest.json to be used by the command. Do you think it would be worthwhile to have a save command or something else to compliment this so we can overwrite in some instances and override in others?

Yes, p12 keys also exist for the serviceAccountFile. I did not implement any handling for that format.